Detectors/TRD/base/include/TRDBase/MCLabel.h

```diff
@@ -19,17 +19,7 @@ namespace o2
 {
 namespace trd
 {
-
-class MCLabel : public o2::MCCompLabel
-{
- public:
-  MCLabel() = default;
-  MCLabel(int trackID, int eventID, int srcID)
-    : o2::MCCompLabel(trackID, eventID, srcID, false) {}
-
-  ClassDefNV(MCLabel, 1);
-};
-
+using MCLabel = o2::MCCompLabel;
```

Comment:
With this change, I am wondering if we really need to keep `o2::trd::MCLabel`. I would suggest removing it. It is more direct and easier to maintain if we just `o2::MCCompLabel` as this is suggesting.